### DISCUSSION TITLE
Automatically move extra family members to a different membership

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#382] [Q&A] [answered]
Hi,

Is there a way to move a family member to a different membership automatically after the main member has signed up. Reason being is i check the membership type to define which price to use in the events. ie the child event price is cheaper than the adult  event price providing they are members. 

Thanks

**Comment by liedekef** [comment, marked as answer]
If you cancel the link between the members, then you can do this but is not automatically.
A familiy membership has only 1 price, so if you want to move family members, you have to do this manually (which is logical: there is no logic in a family membership across memberships, it is simply not possible).
